pkgs/applications/blockchains/daedalus/default.nix

```diff
@@ -0,0 +1,8 @@
+{ fetchFromGitHub, lib }:
+
+(import (fetchFromGitHub {
+  owner = "input-output-hk";
+  repo = "daedalus";
+  rev = "1ae99357a794fb73e78852abc984c3e5cd37d6bf";
+  sha256 = "QBYvDDsvLnWx0mv9wCLtM0RkUEaSuGFXnimR0v+ciFI=";
+}) { }).daedalus
```

Comment:
IFD isn't allowed in nixpkgs

Reply:
Why? Do you prefer I copy-paste the Nix code of this repo into nixpkgs? What good would that do?

Reply:
Look at the ofborg failure, IFD isn't allowed because it dramatically increases evaluation time

Reply:
https://gist.github.com/GrahamcOfBorg/9e0d59aac812189ea3f937d94252cfbf

Reply:
```
error: while evaluating anonymous function at /var/lib/ofborg/checkout/repo/38dca4e3aa6bca43ea96d2fcc04e8229/mr-est/ofborg-evaluator-6/.gc-of-borg-outpaths.nix:47:12, called from undefined position:
while evaluating 'g' at /var/lib/ofborg/checkout/repo/38dca4e3aa6bca43ea96d2fcc04e8229/mr-est/ofborg-evaluator-6/lib/attrsets.nix:301:19, called from undefined position:
while evaluating anonymous function at /var/lib/ofborg/checkout/repo/38dca4e3aa6bca43ea96d2fcc04e8229/mr-est/ofborg-evaluator-6/pkgs/top-level/release-lib.nix:146:38, called from undefined position:
while evaluating 'isDerivation' at /var/lib/ofborg/checkout/repo/38dca4e3aa6bca43ea96d2fcc04e8229/mr-est/ofborg-evaluator-6/lib/attrsets.nix:330:18, called from /var/lib/ofborg/checkout/repo/38dca4e3aa6bca43ea96d2fcc04e8229/mr-est/ofborg-evaluator-6/pkgs/top-level/release-lib.nix:147:10:
while evaluating 'callPackageWith' at /var/lib/ofborg/checkout/repo/38dca4e3aa6bca43ea96d2fcc04e8229/mr-est/ofborg-evaluator-6/lib/customisation.nix:117:35, called from /var/lib/ofborg/checkout/repo/38dca4e3aa6bca43ea96d2fcc04e8229/mr-est/ofborg-evaluator-6/pkgs/top-level/all-packages.nix:2655:14:
while evaluating 'makeOverridable' at /var/lib/ofborg/checkout/repo/38dca4e3aa6bca43ea96d2fcc04e8229/mr-est/ofborg-evaluator-6/lib/customisation.nix:67:24, called from /var/lib/ofborg/checkout/repo/38dca4e3aa6bca43ea96d2fcc04e8229/mr-est/ofborg-evaluator-6/lib/customisation.nix:121:8:
while evaluating anonymous function at /var/lib/ofborg/checkout/repo/38dca4e3aa6bca43ea96d2fcc04e8229/mr-est/ofborg-evaluator-6/pkgs/applications/blockchains/daedalus/default.nix:1:1, called from /var/lib/ofborg/checkout/repo/38dca4e3aa6bca43ea96d2fcc04e8229/mr-est/ofborg-evaluator-6/lib/customisation.nix:69:16:
cannot import '/nix/store/i2kka0h21x7yfyywz8c7asj4a6zbvsfs-source', since path '/nix/store/qw3acxdan1mz00ihz17ff2km2kvdn5m3-source.drv' is not valid, at /var/lib/ofborg/checkout/repo/38dca4e3aa6bca43ea96d2fcc04e8229/mr-est/ofborg-evaluator-6/pkgs/applications/blockchains/daedalus/default.nix:3:2
```

Reply:
upstream can be "nixpkgs-ified", and that's acceptable

Reply:
@jonringer what should I do? I don't understand the direction to get this working. Obviously, copy-paste is not the way.

Reply:
Daedalus uses haskell.nix, so IIUC we would have to replace that with plain Nixpkgs - which would be a significant undertaking I believe.

Alternatively, we could create a derivation that just wraps their pre-built binaries at https://daedaluswallet.io/en/download/ ... just like we do for some non-open-source apps like http://obsidian.md and proprietary browsers like Opera and Vivaldi.